Tests/DependencyInjection/FOSUserExtensionTest.php

```diff
@@ -326,6 +329,66 @@ public function userManagerSetFactoryProvider()
         );
     }
 
+    /**
+     * @dataProvider loadRoutingProvider
+     */
+    public function testLoadRouting($routeName, $path, array $methods)
```

Comment:
> a new testcase dedicated to checking the routing files
And in which path ? Like the configuration path? (`Resources/config`)

I didn't included `all.xml` because this call `@Resources` annotation that are not working whit my simple tests.

btw, `group.xml` is not included on `all.xml`.